app/Resources/views/home/index.html.twig

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 <main>
     <section class="l__wrapper">
-        <div class="l__row l__tablet--col l__row--v-stretch b__nudge--bottom">
+        <div class="l__row l__tablet--col l__row--v-stretch">
```

Comment:
Same here, can you put `{% if not enable_canary %} b__nudge--bottom{% endif %}` so we can disable it in prod while it's not ready but still merge this PR?